catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/commands/SearchFormsManageCommand.java

```diff
@@ -0,0 +1,172 @@
+import ddf.catalog.source.SourceUnavailableException;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.Option;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.catalog.ui.util.EndpointUtil;
+import org.codice.ddf.commands.catalog.SubjectCommands;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Service
+@Command(
+  scope = "forms",
+  name = "manage",
+  description = "Provides capability of viewing and deleting system-templates"
+)
+public class SearchFormsManageCommand extends SubjectCommands {
+
+  @Reference EndpointUtil endpointUtil;
+
+  @Reference CatalogFramework catalogFramework;
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(SearchFormsLoaderCommand.class);
+
+  private final PrintStream console = System.out;
+
+  @Option(
+    name = "--list",
+    required = false,
+    aliases = {"-l"},
+    multiValued = false,
+    description = "List all the system forms before removal"
+  )
+  protected boolean viewAll = false;
+
+  @Option(
+    name = "--remove-single",
+    required = false,
+    aliases = {"-s"},
+    multiValued = false,
+    description = "Remove a single system template by metacard ID"
+  )
+  protected String singleDeletion = null;
+
+  @Option(
+    name = "--remove-all",
+    required = false,
+    aliases = {"-r"},
+    multiValued = false,
+    description = "Removes all the system templates"
+  )
+  protected boolean massDeletion = false;
+
+  @Override
+  protected Object executeWithSubject() {
+
+    if (viewAll) {
+      printSystemTemplates();
+      return null;
+    }
+
+    if (massDeletion || !isEmpty(singleDeletion)) {
+      executeDeletion();
+      return null;
+    }
+
+    return null;
+  }
+
+  private void printSystemTemplates() {
+    for (Metacard mc : retrieveSystemMetacards()) {
+      console.println(
+          ansi()
+              .fgBrightYellow()
+              .a("Title: " + mc.getTitle())
+              .newline()
+              .fgGreen()
+              .a("\t- " + mc.getId())
+              .reset());
+    }
+  }
+
+  private void executeDeletion() {
+
+    List<Metacard> allSystemTemplates = retrieveSystemMetacards();
+    if (allSystemTemplates.isEmpty()) {
+      console.println(ansi().fgBrightRed().a("No system forms exist.").reset());
+      return;
+    }
+
+    try {
+      List<String> systemTempIds =
+          singleDeletion != null
+              ? ImmutableList.of(singleDeletion)
+              : retrieveListMetacardIds(allSystemTemplates);
+
+      List<String> deletedMetacardIds = this.executeSystemTemplateRemoval(systemTempIds);
+
+      if (deletedMetacardIds.isEmpty()) {
+        console.println(ansi().fgBrightRed().a("No system forms were deleted").reset());
```

Comment:
✏️ I would recommend pulling the brightRed and brightYellow chains to two functions called warn and error.